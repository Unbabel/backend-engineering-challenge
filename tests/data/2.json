{"timestamp": "2020-01-01 01:11:08.509654","translation_id": "6aa5b2f39f7254a75aa5","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 200}
{"timestamp": "2020-01-01 01:12:19.903159","translation_id": "6aa5b2f39f7354a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 30}
{"timestamp": "2020-01-01 01:13:19.903159","translation_id": "6aa5b2f39f7454a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 150}
{"timestamp": "2020-01-01 01:14:19.903159","translation_id": "6aa5b2f39f7554a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 100}
{"timestamp": "2020-01-01 01:15:19.903159","translation_id": "6aa5b2f39f7654a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 10}
{"timestamp": "2020-01-01 01:16:19.903159","translation_id": "6aa5b2f39f7754a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 22}
{"timestamp": "2020-01-01 01:17:19.903159","translation_id": "6aa5b2f39f7854a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 56}
{"timestamp": "2020-01-01 01:18:19.903159","translation_id": "6aa5b2f39f7954a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 49}
{"timestamp": "2020-01-01 01:19:19.903159","translation_id": "6aa5b2f39f7a54a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 9800}
{"timestamp": "2020-01-01 01:20:19.903159","translation_id": "6aa5b2f39f7b54a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 780}
{"timestamp": "2020-01-01 01:21:19.903159","translation_id": "6aa5b2f39f7c54a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 10}
{"timestamp": "2020-01-01 01:21:19.903159","translation_id": "6aa5b2f39f7d54a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 20}