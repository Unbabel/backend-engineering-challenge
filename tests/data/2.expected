{"date": "2020-01-01 01:11:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:12:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:13:00", "average_delivery_time": 115.0}
{"date": "2020-01-01 01:14:00", "average_delivery_time": 126.66666666666667}
{"date": "2020-01-01 01:15:00", "average_delivery_time": 120.0}
{"date": "2020-01-01 01:16:00", "average_delivery_time": 98.0}
{"date": "2020-01-01 01:17:00", "average_delivery_time": 85.33333333333333}
{"date": "2020-01-01 01:18:00", "average_delivery_time": 81.14285714285714}
{"date": "2020-01-01 01:19:00", "average_delivery_time": 77.125}
{"date": "2020-01-01 01:20:00", "average_delivery_time": 1157.4444444444443}
{"date": "2020-01-01 01:21:00", "average_delivery_time": 1119.7}
{"date": "2020-01-01 01:22:00", "average_delivery_time": 1102.7}
